grafana/README.md

```diff
@@ -0,0 +1,22 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster.
+* single: dashboards for a single resource(e.g. `node` or `pod`).
+
+**Notice**
+* summary dashboards use `alias` as the `cluster` template variable.
```

Comment:
@brancz Yep. It is the user's choice of whether `external_labels` or target level custom `cluster` label is fine to them. 

Finally, we make a conclusion that we should use a `cluster` template var.

I have updated the `node.json` file and readme. Could you please take a look at this.